upload/upload.go

```diff
@@ -193,6 +187,21 @@ func NewHandler(
 			B64Identity: b64Identity,
 		}
 
+		var maxSize int64
+
+		switch vr.Service {
+		case "qpc":
```

Comment:
Is this a temporary approach (to get the customer moving)?  Couple of things I see that make me a little uneasy about this:  1) you are now moving knowledge of a service/customer/client ("qpc") into the ingress service  2) if we need to increase the size again for qpc, then we have to recompile, re-release.

Have you thought about using hash map for this type of configuration?  That way you could configure a max size per service type outside of the code.  The hash map might look something like this:
```
{"qpc": 150, "advisor": 100}
```
If the service is not defined in the hash map, then you could fall back to using the default max size.

Just a thought...

Reply:
@dehort I took another stab at it with the map method. Let me know if this looks better.